cypress/support/index.js

```diff
@@ -0,0 +1,37 @@
+import { addMatchImageSnapshotCommand } from "cypress-image-snapshot/command";
+
+addMatchImageSnapshotCommand({
+  failureThreshold: 0,
+  failureThresholdType: "percent",
+});
+
+// https://stackoverflow.com/questions/521295/seeding-the-random-number-generator-in-javascript/47593316#47593316
+const LCG = seed => () =>
+  ((2 ** 31 - 1) & (seed = Math.imul(48271, seed))) / 2 ** 31;
+
+const MathRandom = Math.random;
+beforeEach(() => {
+  Math.random = LCG(0);
```

Comment:
Could this use `jest.fn` somehow? or does it just make sense to modify the environment directly?

Reply:
jest.fn is useful if you want to assert that the function has been called. I don't think it's super useful here.